tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -7,6 +7,8 @@ Image:
         Encrypted: True
 
 Build:
+    Imds:
+        RequireImdsV2: true
```

Comment:
Just for my better understanding: since we are enforcing IMDSv2 at conftest level, why should we explicitly set this to true here?

Reply:
It is not needed, I did it to be explicit that we were testing the positive case.